Gems/Atom/Tools/ShaderManagementConsole/Code/Source/Window/ShaderManagementConsoleTableView.cpp

```diff
@@ -26,13 +29,15 @@ namespace ShaderManagementConsole
         setSelectionBehavior(QAbstractItemView::SelectItems);
         setSelectionMode(QAbstractItemView::SingleSelection);
         horizontalHeader()->setSectionResizeMode(QHeaderView::ResizeToContents);
+        connect(horizontalHeader(), &QHeaderView::sectionDoubleClicked, this, &ShaderManagementConsoleTableView::OnOptionClicked);
```

Comment:
I'd agree here with the comment from @gadams3 , this should be a right-click with a popup menu. Also the dialog box should show the material count total.